src/test/java/com/saebyeok/saebyeok/service/ArticleServiceTest.java

```diff
@@ -163,4 +160,24 @@ void readMemberArticleTest() {
         assertThat(articleResponse.getLikesCount()).isNotNull();
         assertThat(articleResponse.getIsLikedByMe()).isNotNull();
     }
+
+    @DisplayName("게시글을 감정에 따라 필터하여 특정 감정의 게시글만 표시한다")
+    @Test
+    void filterArticleTest() {
+        when(articleEmotionService.findArticlesByEmotionIds(any(), any(), any())).thenReturn(Arrays.asList(article1, article2));
+
+        List<ArticleResponse> memberArticles = articleService.getMemberArticles(member, 1, 5, Arrays.asList(1L, 2L));
+        assertThat(memberArticles.size()).isEqualTo(2);
```

Comment:
```suggestion
        assertThat(memberArticles).hasSize(2);
```
리스트 크기 비교는 hasSize가 더 명확하지 않을까 싶네용! :)